source/class/qx/tool/compiler/Analyser.js

```diff
@@ -1415,6 +1415,16 @@ qx.Class.define("qx.tool.compiler.Analyser", {
      * @param library
      */
     addLibrary(library) {
+      if (this.__librariesByNamespace[library.getNamespace()]) {
+        throw new Error(
+          "Multiple libraries with namespace " +
+            library.getNamespace() +
+            " found " +
+            library.getRootDir() +
+            " and " +
+            this.__librariesByNamespace[library.getNamespace()].getRootDir()
```

Comment:
The same about `this.__librariesByNamespace[library.getNamespace()]`. You called it twice.

Reply:
IMHO it could be at it is....

Reply:
yes, i agree,  the `.getNamespace` is only a property